Formula/ddns-go.rb

```diff
@@ -0,0 +1,23 @@
+class DdnsGo < Formula
+  desc "Simple and easy-to-use DDNS"
+  homepage "https://github.com/jeessy2/ddns-go"
+  url "https://github.com/jeessy2/ddns-go/archive/refs/tags/v5.3.1.tar.gz"
+  sha256 "afca100b1b2d2b5bdcb0f6d061709136a567978e9cca85b9c0867d9219ead3f5"
+  license "MIT"
+  head "https://github.com/jeessy2/ddns-go.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -X main.version=v#{version}
+      -X main.buildTime=#{time.iso8601}
+      -extldflags -static -s -w
+    ]
+    system "go", "build", *std_go_args(ldflags: ldflags), "-trimpath"
```

Comment:
```suggestion
    system "go", "build", *std_go_args(ldflags: ldflags)
```
This is already part of `std_go_args`.

Reply:
Fixed.